update/sync/syncTransactions.go

```diff
@@ -165,16 +165,13 @@ func (ts *transactionsSync) requestTransactionsFor(miniBlock *block.MiniBlock) i
 	switch mbType {
 	case block.TxBlock:
 		ts.requestHandler.RequestTransaction(miniBlock.SenderShardID, missingTxs)
-		// TODO - uncomment this
-		// ts.requestHandler.RequestTransaction(miniBlock.ReceiverShardID, missingTxs)
+		ts.requestHandler.RequestTransaction(miniBlock.ReceiverShardID, missingTxs)
 	case block.SmartContractResultBlock:
 		ts.requestHandler.RequestUnsignedTransactions(miniBlock.SenderShardID, missingTxs)
-		// TODO - uncomment this
-		// ts.requestHandler.RequestUnsignedTransactions(miniBlock.ReceiverShardID, missingTxs)
+		ts.requestHandler.RequestUnsignedTransactions(miniBlock.ReceiverShardID, missingTxs)
 	case block.RewardsBlock:
 		ts.requestHandler.RequestRewardTransactions(miniBlock.SenderShardID, missingTxs)
-		// TODO - uncomment this
-		// ts.requestHandler.RequestRewardTransactions(miniBlock.ReceiverShardID, missingTxs)
+		ts.requestHandler.RequestRewardTransactions(miniBlock.ReceiverShardID, missingTxs)
```

Comment:
Why is this component used for bootstrap now ? In case of update - there is a different process.

Reply:
In case of hard fork - this helps the requesting and getting of data.

Reply:
for bootstrap with scheduled active, we need to request txs from last blocks

Reply:
Even in case of shuffle out without scheduled we request tes from last blocks.